yogstation/code/game/objects/items/plushes.dm

```diff
@@ -138,7 +138,6 @@
 		new/obj/item/toy/plush/goatplushie/angry/guardgoat(location)
 
 /obj/item/toy/plush/goatplushie/angry/kinggoat/attackby(/obj/item/reagent_containers/food/snacks/grown/cabbage/C,mob/living/user,params)
```

Comment:
cant you shove malformed params in here? Sus!

Reply:
Oh wait

so with that slash there, I think this becomes a global var: https://www.byond.com/forum/post/2830750